Sources/Orbit/Foundation/CountryFlags/CountryFlags.swift

```diff
@@ -0,0 +1,264 @@
+        case `do`
+        case dz
+        case ec
+        case ee
+        case eg
+        case eh
+        case er
+        case es
+        case et
+        case eu
+        case fi
+        case fj
+        case fk
+        case fm
+        case fo
+        case fr
+        case ga
+        case gb
+        case gd
+        case ge
+        case gf
+        case gg
+        case gh
+        case gi
+        case gl
+        case gm
+        case gn
+        case gp
+        case gq
+        case gr
+        case gs
+        case gt
+        case gu
+        case gw
+        case gy
+        case hk
+        case hm
+        case hn
+        case hr
+        case ht
+        case hu
+        case ic
+        case id
+        case ie
+        case il
+        case im
+        case `in`
+        case io
+        case iq
+        case ir
+        case `is`
+        case it
+        case je
+        case jm
+        case jo
+        case jp
+        case ke
+        case kg
+        case kh
+        case ki
+        case km
+        case kn
+        case kp
+        case kr
+        case kw
+        case ky
+        case kz
+        case la
+        case lb
+        case lc
+        case li
+        case lk
+        case lr
+        case ls
+        case lt
+        case lu
+        case lv
+        case ly
+        case ma
+        case mc
+        case md
+        case me
+        case mf
+        case mg
+        case mh
+        case mk
+        case ml
+        case mm
+        case mn
+        case mo
+        case mp
+        case mq
+        case mr
+        case ms
+        case mt
+        case mu
+        case mv
+        case mw
+        case mx
+        case my
+        case mz
+        case na
+        case nc
+        case ne
+        case nf
+        case ng
+        case ni
+        case nl
+        case no
+        case np
+        case nr
+        case nu
+        case nz
+        case om
+        case pa
+        case pe
+        case pf
+        case pg
+        case ph
+        case pk
+        case pl
+        case pm
+        case pn
+        case pr
+        case ps
+        case pt
+        case pw
+        case py
+        case qa
+        case re
+        case ro
+        case rs
+        case ru
+        case rw
+        case sa
+        case sb
+        case sc
+        case sd
+        case se
+        case sg
+        case sh
+        case si
+        case sj
+        case sk
+        case sl
+        case sm
+        case sn
+        case so
+        case sr
+        case ss
+        case st
+        case sv
+        case sx
+        case sy
+        case sz
+        case tc
+        case td
+        case tf
+        case tg
+        case th
+        case tj
+        case tk
+        case tl
+        case tm
+        case tn
+        case to
+        case tr
+        case tt
+        case tv
+        case tw
+        case tz
+        case ua
+        case ug
+        case um
+        case unknown
```

Comment:
I'd probably keep this `case unknown` as the first member in the enumeration, as a good practise. 🤔